scripts/envSettings.js

```diff
@@ -1,13 +0,0 @@
-const fsExtra = require('fs-extra');
-
-function createEnvSettingsFile(settingsFile) {
```

Comment:
Hvorfor fjerner vi denne? :D

Reply:
kunne ikke se at den blir brukt?

Reply:
har sett om den importeres noe sted + logget console.log(window) for å se om den brukes